index.html

```diff
@@ -11,34 +11,217 @@
+                            </ul>  
+                        </div>
+                    </div>  
+                </div>
+            </div>
+        </div>
+        <section class="about-container">
+            <div class="first-about-container">
+                <h1 class="about-header">About me</h1>
+                <p class="about-para">Hello I’m a software developer! I can help you build a product , 
+                    feature or website Look through some of my work and experience! 
+                    If you like what you see and have a project you need coded, don’t hestiate to contact me.
+                </p>
+                <a href="#"><button class="about-btn-text hover-effect">Get My Resume</button></a>
+                <img class="about-illu" src="./img/about/illustration-about-me-mobile.svg" alt="illustration">
+            </div>
+            <img class="about-divider" src="./img/about/Vector 4.svg" alt="">
+            <div class="skills-container">
+                <div class="language">
+                    <ul class="language-header">
+                        <li><img src="./img/about/Rectangle 55.svg" alt="Rectangle"></li>
+                        <li class="language-h1">Language</li>
+                    </ul>
+                    <ul class="main-language">
+                        <li class="skill-js">JavaScript</li>
+                        <li class="skill-rub">Ruby</li>
+                        <li class="skill-html">Html</li>
+                        <li class="skill-css">Css</li>
+                    </ul>
+                </div>
+                <div class="framework">
+                    <ul class="framework-header">
+                        <li><img src="./img/about/Rectangle 56.svg" alt="Rectangle"></li>
+                        <li class="framework-h1">Frameworks</li>
+                    </ul>
+                    <ul class="main-framework">
+                        <li class="frame-boot">Bootstrap</li>
+                        <li class="frame-rails">Ruby on Rails</li>
+                        <li class="frame-rspec">RSPec</li>
+                        <li class="frame-capy">CapyBara</li>
+                        <li class="frame-sel">Selenium</li>
+                    </ul>
+                </div>
+                <div class="skills">
+                    <ul class="skills-header">
+                        <li><img src="./img/about/Rectangle 57.svg" alt="Rectangle"></li>
+                        <li class="skills-h1">Skills</li>
+                    </ul>
+                    <ul class="main-skills">
+                        <li class="skill-codekit">Codekit</>
+                        <li class="skill-github">Github</li>
+                        <li class="skill-codepen">Codepen</li>
+                        <li class="skill-gitlab">Gitlab</li>
+                        <li class="skill-terminal">Terminal</li>
+                    </ul>
+                </div>  
+            </div>
```

Comment:
Please consider following these 2 rules in this section as it's a good practice for semantics:-

1. The technology categories ("Languages", "Frameworks" and "Skills") are in a `ul`, each of them in a `li`
2. The technology in each category(e.g., "JavasScript", "HTML" and "CSS") are also a list `ul` and each of them a `li`


## Solution with detailed demonstration:-

Following is the code representation of the above if you are confused about what the instructions mean:-


```
    <ul>
      <li>
        Languages
        <ul>
          <li>JavaScript</li>
          <li>Ruby</li>
          <li>Html</li>
          <li>CSS</li>
        </ul>
      </li>
      <li>
        Frameworks
        <ul>
          <li>Bootstrap</li>
          <li>Ruby on Rails</li>
          <li>Rspec</li>
          <li>Capybara</li>
          <li>Selenium</li>
        </ul>
      </li>
      <li>
        Skills
        <ul>
          <li>Codekit</li>
          <li>GitHub</li>
          <li>Code Pen</li>
          <li>Gitlab</li>
          <li>Terminal</li>
        </ul>
      </li>
    </ul>
```